2.1-Segwit-Version-1.ipynb

```diff
@@ -411,17 +410,20 @@
     "pubkey1 = privkey1.get_pubkey()\n",
     "pubkey2 = privkey2.get_pubkey()\n",
     "\n",
-    "# Generate MuSig key\n",
-    "# Method: use generate_musig_key()\n",
-    "\n",
+    "# Generate a 2-of-2 aggregate MuSig key\n",
+    "# Method: generate_musig_key([ECPubKey_list])\n",
```

Comment:
nit, I would drop the square parenthesis, I think the variable name tells enough.

Reply:
Thx. Fixed